anago

```diff
@@ -742,6 +743,23 @@ get_build_candidate () {
                                 $PARENT_BRANCH \
    || return 1
 
+  # When creating alphas, make sure that X.X.0 has been created first or warn.
```

Comment:
I think we're already abusing the branch-early-and-branchff-endlessly model.  I believe this was originally a stopgap back in the 1.2 timeframe, but it's become more regular.  I think we should be branching later and generally discouraging this behavior anyway. WDYT @kubernetes/kubernetes-release-managers?